extensions/wikia/Email/Email.i18n.php

```diff
@@ -3,6 +3,7 @@
 
 $messages['en'] = array(
 	'emailext-desc' => 'An extension to handle delivering email',
+	'emailext-anonymous-editor' => 'A Wikia fan',
```

Comment:
@mklucsarits good point.  I'm on the edge of whether to do it though since it seems like she may want two different behaviors out this string which I'm inclined to handle separately.  I'll see what she says.